doc/modules/naive_bayes.rst

```diff
@@ -222,6 +222,36 @@ It is advisable to evaluate both models, if time permits.
    <http://citeseerx.ist.psu.edu/viewdoc/summary?doi=10.1.1.61.5542>`_
    3rd Conf. on Email and Anti-Spam (CEAS).
 
+.. _categorical_naive_bayes:
+
+Categorical Naive Bayes
+-----------------------
+
+:class:`CategoricalNB` implements the categorical naive Bayes 
+algorithm for categorically distributed data. It assumes that each feature, 
```

Comment:
Is there a reference for this?

Reply:
I didn't find any. CategoricalNB can be seen as a generalization of Bernoulli Naive Bayes if that is of any help.